src/main/java/org/zalando/nakadi/domain/ResourceImpl.java

```diff
@@ -40,6 +40,9 @@ public String getType() {
     @Override
     public Optional<List<AuthorizationAttribute>> getAttributesForOperation(
             final AuthorizationService.Operation operation) {
+        if (null == authorization) {
```

Comment:
is it a real case that authorization is null?
In `ResourceImpl` constructor it is passed as one of parameters, so if we expect null as one of possible values - it should be marked as `@Nullable` in constructor parameter.

Reply:
Yes, `authorization` can be `null`, when "authorization" section is not specified in the request when creating an event-type or subscription. I added the annotation as you suggested to fix it.

Reply:
But I suggested to have annotation in ResourceImpl constructor.